docs/source/challenge-doc.rst

```diff
@@ -144,9 +144,8 @@ or directly from the simulator might not be accessible during submission evaluat
         - Goal   = value[3]
         - The rest = value[4]
 
-    - Finger tip location is accessed by an index i for each finger tips. The index i is in [0-4] and the location of finger 0 (thumb) can acccessed by obs_dict['fin0']
     
-    - Start and Goal positions are 2 dimensional because the height of the two pillars will be constant
+    - Start and Goal positions are 3 dimensional but the height of the two pillars are the same
```

Comment:
how is possible that the pillar high is the same if the high changes in the object? When is higher than the pillar it would then fall on the pillar?

Reply:
Good point! The original idea is that the height of the pillar is a fixed constant, and the object is spawned on the pillar. @Balint-H Is the change in object shape takes into account that the pillar height is constant?

Reply:
The pillar height changes as well, and the object follows along at a fixed vertical offset of 10 cm. This does mean at the start the object falls a small amount. The height offset at the moment doesn't take into account the object height change, so the height range should not be increased beyond touching the pillar, otherwise it would start from penetration with the pillar.

Reply:
To summarise, there is a small change to accommodate the changing object size. Doc updated in [commit](https://github.com/MyoHub/myosuite/commit/d192bd93b0f94754cf9a9f4940b77a6e0c21c29a)